### PR TITLE
Updated documentation in For Ubuntu Linux and Docker sections

### PR DESCRIPTION
**Pull request by rabadashTheFool**
Updated documentation in For Ubuntu Linux and Docker sections

WE2-572

Signed-off-by: Mihkel Kivisild <mihkel.kivisild@hotmail.com>

### CODE REVIEW

**Comment by rabadashTheFool** [comment]
Some documentation changes were already made in #21 .

File: README.md [hunk 1]
```diff
@@ -36,16 +39,52 @@ The algorithm, which performs the validation of the Web eID authentication token
...
+    ```
+4. Copy the necessary DigiDoc C# library files into your project:
+
+    ```sh
+    cp /usr/include/digidocpp_csharp/* WebEid.AspNetCore.Example/DigiDoc/
+    ```
+
+#### For macOS
+
+1.  Install the _libdigidocpp-3.14.4.pkg_ package or higher. The installation packages are available from  [https://github.com/open-eid/libdigidocpp/releases](https://github.com/open-eid/libdigidocpp/releases).
```
Comment (metsma) [inline README.md:80]: I don't think you can sign with 3.14.4 version at all today

Comment (rabadashTheFool) [reply README.md:80]: Should we update it to the latest available release then?
Like so:
"Install the _libdigidoc_cpp-3.17.1.pkg_ package or higher.

And then update it for Windows as well.

Comment (metsma) [reply README.md:80]: Probably yes

Comment (rabadashTheFool) [reply README.md]: I have updated the version number.

**Review by metsma — APPROVED** [review]

**Pull request merged**